### ISSUE TITLE
Regression: Lower performance on SSX after 0.0.7-8934 (#6790)

### ISSUE THREAD

**Issue by jeremybris** [RPCS3/rpcs3#9776]
## Quick summary
On and after RPCS3 0.0.7-8934, PR #6790, the framerate is goes from ~30 F.P.S. to ~20 F.P.S.

## Details

At the start of an event, before you press "X" to go downhill, the F.P.S. is lower. Here is [my save](https://github.com/RPCS3/rpcs3/files/5831398/SSX.Save.zip) if you need it.

I can not attach logs because they are over 3 gigabytes each. But if you need them, I will upload them to Google drive.

| RPCS3 Version | RPCS3 Commit | RPCS3 FPS & log | Screenshot |
| ------------- | ------------- | ------------- | ------------- |
| v0.0.7-8934 and Before  | -------------  | ~30 F.P.S. | ![Capture157](https://user-images.githubusercontent.com/11889359/107857933-54b26e80-6dff-11eb-838f-5ecb6377fd25.PNG) |
| v0.0.7-8934 and After | #8710  | ~20 F.P.S. (Regression) | ![Capture345](https://user-images.githubusercontent.com/11889359/107857929-48c6ac80-6dff-11eb-9e3a-82a79a8a8683.PNG) |

#### 6. Please provide your system configuration:
* OS - Windows 10, Latest Version
* CPU - Ryzen 9
* GPU - RTX 2000
* Settings - Default Settings with Resolution 450 %

**Comment by Asinin3** [comment]
450% res scale isn't good, a proper integer like 400% would be better, but even still SSX doesn't play kindly with scaling. Even a RTX 3080 can struggle with 200% in this game on some stages. I would stick with 200%.

**Comment by kd-11** [comment]
SSX demands a lot of reconstruction during rendering and the steps can be very expensive. This increase in cost is exponential as you increase res. Optimizations can be made to ease the burden, but this is very low priority considering the other ~600 open issues.

**Comment by jeremybris** [comment]
- Tested before 0.0.11-10743, another F.P.S. regression, and it was present. So, I think this is still present in master. 
- Running at lower resolutions seems to fix this, or makes it unnoticeable. It's a good workaround. 

SSX has some resolution scaling problem(s). ~20 F.P.S. loss, with no noticeable visual difference, seems significant to me. If you choose, you can close this with that workaround. If not, at the least, I hope someone can make a patch. I understand it is low priority, and I don't mean to impose.

**Comment by Yahfz** [comment]
What's your GPU? Like we mentioned above even an RTX3080 has issues at 200%, I'm sure that even if you had a 2080Ti oced you'd have issues.

**Comment by jeremybris** [comment]
RTX 2080 Super. I experience F.P.S. loss in upscaling in other games, but not like this. The "eating performance" is not a problem; it's the "eating more performance than it did before", and the visuals seem the same. **The game is now unplayable, it crashes, at 450%, when it used to be playable.** Furthermore, it's the only game that does this, for me. But forgive me, if that is how the emulator is supposed to be.

**Comment by FlexBy420** [comment]
Is this still an issue?

**Comment by jeremybris** [comment]
I think so, performance is still noticibly bad in upscaling, compared to other titles, as of .33-17046 (10/19/2024). It used to be more steady; now it dips to 25 fps at 4K. In the example, 450％ scale is used to show the semicubical parabola of f.p.s. when scaling.

It may not necessarily be a fault of the emulator, but the game; I'm just hoping someone can make a patch for this game using the above information to make it more playable in 4K. :)

**Comment by digant73** [comment]
@jeremybris a lot of changes have been made in the last year. Please retest

**Comment by jeremybris** [comment]
> [@jeremybris](https://github.com/jeremybris) a lot of changes have been made in the last year. Please retest

Yes, the game still dips hard, into the low 20 F.P.S. in 4K, making the game unplayable. Hoping someone can make a patch with the information above.

<img width="3839" height="2159" alt="Image" src="https://github.com/user-attachments/assets/3c665b4a-253b-4350-8843-bf183fe0c3c6" />

<img width="3786" height="2159" alt="Image" src="https://github.com/user-attachments/assets/7bdeadf4-7544-4704-a920-2d62ff91a20f" />

My specs are 5070 ti, AMD 7900 (non-x)

**Comment by kd-11** [comment]
Might be worth taking another look at this one after so long. Something is wrong here, the gap in CPU and GPU should have made the dips disappear by now.

**Comment by kd-11** [comment]
Partly fixed by https://github.com/RPCS3/rpcs3/pull/17676
This ticket can be closed, the real cause is already identified in this comment: https://github.com/RPCS3/rpcs3/issues/8294#issuecomment-3507096922
